project/Versions.scala

```diff
@@ -2,7 +2,9 @@
 object Versions {
   val cpg           = parseVersion("cpgVersion")
   val js2cpg        = parseVersion("js2cpgVersion")
-  val antlr         = "4.10"
+  // Dont upgrade antlr to 4.10 or above since those versions require java 11 or higher which
```

Comment:
Then we perhaps want to add it to: https://github.com/joernio/joern/blob/master/.scala-steward.conf?

Reply:
Good point. Separate PR https://github.com/joernio/joern/pull/1281